modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/utilities/index.js

```diff
@@ -14,6 +14,20 @@
 
 import {fetch} from 'frontend-js-web';
 
+import createOdataFilter from './odata';
+
+export function getData(apiUrl, query) {
+	let url = apiUrl;
+
+	if (query) {
+		url += (url.includes('?') ? '&' : '?') + `search=${query}`;
+	}
+
+	return fetch(url, {
```

Comment:
We could avoid this manual string inspection by using `URL` and `searchParams`, I think.

Something like:

```js
const url = new URL(apiUrl);

if (query) {
  url.searchParams.append('search', query);
}

return fetch(url, {
    // ...
```